docs/zh-cn/cli_commands.md

```diff
@@ -0,0 +1,442 @@
+
+**用法**：
+
+```
+qmk c2json -km KEYMAP -kb KEYBOARD [-q] [--no-cpp] [-o OUTPUT] filename
+```
+
+## `qmk lint`
+
+检查键盘及键映射数据并提示出常见错误与问题，以及不符合模板规范的地方。
+
+**用法**：
+
+```
+qmk lint [-km KEYMAP] [-kb KEYBOARD] [--strict]
+```
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**示例**：
+
+基本的lint检查：
+
+    qmk lint -kb rominronin/katana60/rev2
+
+## `qmk list-keyboards`
+
+该命令可以列出 `qmk_firmware` 中所有的键盘
+
+**用法**：
+
+```
+qmk list-keyboards
+```
+
+## `qmk list-keymaps`
+
+该命令可以列出指定键盘（及指定版本）下的所有键映射。
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**用法**：
+
+```
+qmk list-keymaps -kb planck/ez
+```
+
+## `qmk new-keyboard`
+
+该命令可基于现有模板创建出新的键盘定义。
+
+对于未给出的参数，会提示你输入，若未传入 `-u` 参数且 .gitconfig 中设置了 `user.name`，则会提示你使用该值作为默认用户名。
+
+**用法**：
+
+```
+qmk new-keyboard [-kb KEYBOARD] [-t {avr,ps2avrgb}] -u USERNAME
+```
+
+## `qmk new-keymap`
+
+该命令可基于键盘已有的默认键映射创建新的键映射。
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**用法**：
+
+```
+qmk new-keymap [-kb KEYBOARD] [-km KEYMAP]
+```
+
+## `qmk clean`
+
+该命令会清理 `.build` 目录，若传入 `--all` 开关，在 `qmk_firmware` 下的所有.hex及.bin文件也会一并删除。
+
+**用法**：
+
+```
+qmk clean [-a]
+```
+
+---
+
+# 面向开发者的命令
+
+## `qmk format-text`
+
+该命令会重新格式化并统一文件的换行符。
+
+代码库下所有的文件须使用Unix换行符（LF）。
+若你在**Windows**下进行开发，必须确保文件中的换行符是正确的，才能让你的PR被允许合入。
+
+```
+qmk format-text
+```
+
+## `qmk format-c`
+
+该命令会使用clang-format来格式化C代码。
+
+不带参数地执行该命令以用来格式化核心代码相关的改动，默认会通过 `git diff` 来检查 `origin/master`， 可以通过 `-b <分支名>` 来改变检查的分支。
+
+带着 `-a` 开关执行命令会格式化所有的核心代码，也可以在命令行中传入文件名来指定格式化某个文件。
+
+**用以处理指定文件时**：
+
+```
+qmk format-c [file1] [file2] [...] [fileN]
+```
+
+**用以处理所有的核心代码时**：
+
+```
+qmk format-c -a
+```
+
+**用以处理 origin/master 下的所有改动时**：
+
+```
+qmk format-c
+```
+
+**用以处理指定分支下的所有改动时**：
+
+```
+qmk format-c -b branch_name
+```
+
+## `qmk docs`
+
+该命令会在本地启动一个HTTP服务，从而你可以浏览及改进文档，默认端口号为8936，使用 `-b`/`--browser` 开关可以让该命令自动通过默认浏览器打开链接地址。
+
+**用法**：
+
+```
+qmk docs [-b] [-p PORT]
+```
+
+## `qmk generate-docs`
+
+该命令可以在本地生成QMK文档，用以文档的常规浏览使用，或进行文档改进工作。可以使用类似[serve](https://www.npmjs.com/package/serve)这样的工具来浏览生成的文档文件。
+
+**用法**：
+
+```
+qmk generate-docs
+```
+
+## `qmk generate-rgb-breathe-table`
+
+该命令可以生成用于[RGB灯光](zh-cn/feature_rgblight.md)的呼吸效果的查询表（LUT）头文件。将该文件命名为 `rgblight_breathe_table.h` 并放入键盘或键映射目录下，可以覆盖替换 `quantum/rgblight/` 下的默认LUT。
+
+**用法**：
+
+```
+qmk generate-rgb-breathe-table [-q] [-o OUTPUT] [-m MAX] [-c CENTER]
+```
+
+## `qmk kle2json`
+
+该命令可以将KLE原始数据转换成QMK配置器的JSON数据，可接受的输入可以是文件绝对路径，或当前目录下的文件名。若 `info.json` 文件存在，默认不会进行覆盖，通过指定 `-f` 或 `--force` 开关可以允许覆盖。
+
+**用法**：
+
+```
+qmk kle2json [-f] <filename>
+```
+
+**示例**：
+
+```
+$ qmk kle2json kle.txt 
+☒ File info.json already exists, use -f or --force to overwrite.
+```
+
+```
+$ qmk kle2json -f kle.txt -f
+Ψ Wrote out to info.json
+```
+
+## `qmk format-python`
+
+该命令可以对 `qmk_firmware` 下的python代码进行格式化。
+
+**用法**：
+
+```
+qmk format-python
+```
+
+## `qmk pytest`
+
+该命令会执行python测试框架，在你更改了python代码后，应确保该命令可以成功执行。
+
+**用法**：
+
+```
+qmk pytest
+```
```

Comment:
Missing examples https://github.com/qmk/qmk_firmware/blob/20beac5e571e3cb7694a53386c5b715d1bf6eee0/docs/cli_commands.md?plain=1#L477-L500

Reply:
added in latest commits